pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandaloneStarter.java

```diff
@@ -240,6 +242,22 @@ void start() throws Exception {
             log.info(e.getMessage());
         }
 
+        // Create a public tenant and default namespace
+        final String publicTenant = TopicName.PUBLIC_PROPERTY;
+        final String defaultNamespace = TopicName.PUBLIC_PROPERTY + "/" + TopicName.DEFAULT_NAMESPACE;
+        try {
+            if (!admin.properties().getProperties().contains(publicTenant)) {
+                admin.properties().createProperty(
+                    publicTenant,
+                    new PropertyAdmin(Sets.newHashSet(config.getSuperUserRoles()), Sets.newHashSet(cluster)));
+            }
+            if (!admin.namespaces().getNamespaces(publicTenant).contains(defaultNamespace)) {
+                admin.namespaces().createNamespace(defaultNamespace);
```

Comment:
For v2, we need to specify the "replication" cluster. I added a convenience method in #1562, though you can just add : 
```java
admin.namespaces().setNamespaceReplicationClusters(defaultNamespace, Sets.newHashSet(config.getClusterName());
```

Reply:
done